src/output/style.rs

```diff
@@ -80,11 +80,15 @@ impl Format {
                 if args.is_null() {
                     for name in names {
                         let (x, _q) = name.evaluate(scope)?;
-                        if let Some((sub_context, file)) =
-                            file_context.find_file(x.as_ref())
+                        if let Some((sub_context, path, mut file)) =
+                            context.find_file(x.as_ref())?
```

Comment:
Not that `find_file` takes an `&str`, I think `x.as_ref()` could be changed to `&x` here?

Reply:
Correct